docs/debug_virt_stack/launch-qemu-gdb.md

```diff
@@ -0,0 +1,187 @@
+# Launch QEMU with gdb and connect locally with gdb client
+
+This guide is for cases where QEMU counters very early failures and it is hard to synchronize it in a later point in time.
+
+## Image creation and PVC population
+
+This scenario is a slight variation of the [guide about starting strace](launch-qemu-strace.md), hence some of the details on the image build and the PVC population are simply skipped and explained in the other section.
+
+In this example, QEMU will be launched with [`gdbserver`](https://man7.org/linux/man-pages/man1/gdbserver.1.html) and later we will connect to it using a local `gdb` client.
+
+The wrapping script looks like:
+```bash
+#!/bin/bash
+
+LD_LIBRARY_PATH=$LD_LIBRARY_PATH:/var/run/debug/usr/lib64 /var/run/debug/usr/bin/gdbserver \
+	localhost:1234 \
+	/usr/libexec/qemu-kvm $@ &
+printf "%d" $(pgrep gdbserver) > /run/libvirt/qemu/run/default_vmi-debug-tools.pid
+
+```
+
+First, we need to build and push the image with the wrapping script and the gdbserver:
+```dockerfile
+FROM quay.io/centos/centos:stream9 as build
+
+ENV DIR /debug-tools
+ENV DEBUGINFOD_URLS https://debuginfod.centos.org/
+RUN mkdir -p ${DIR}
+RUN mkdir -p ${DIR}/logs
+
+RUN yum  install --installroot=${DIR} -y gdb-gdbserver && dnf clean all
```

Comment:
for consistency: shall we use yum or dnf?

Reply:
Thanks! I'll change this